src/gluonts/evaluation/metrics.py

```diff
@@ -87,7 +87,7 @@ def coverage(target: np.ndarray, forecast: np.ndarray) -> float:
 
         coverage = mean(Y < \hat{Y})
     """
-    return np.mean(target < forecast)
+    return float(np.mean(target < forecast))
```

Comment:
Just wondering: is this related?

Reply:
This was somehow necessary to make the mypy check happy. No idea why -- probably some change in numpy?